### PR TITLE
Initial chart

### PR DESCRIPTION
**Pull request by QuentinBisson**
Adds kube eagle as an app

### CODE REVIEW

File: helm/kube-eagle/templates/_helpers.tpl [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+giantswarm.io/service-type: {{ .Values.serviceType }}
+{{- end -}}
+
+{{/*
+Selector labels
+*/}}
+{{- define "labels.selector" -}}
+app.kubernetes.io/name: {{ include "kube-eagle.name" . | quote }}
+app.kubernetes.io/instance: {{ .Release.Name | quote }}
+{{- end -}}
```
Comment (rossf7) [inline helm/kube-eagle/templates/_helpers.tpl]: Nitpick but CI might fail due to the line endings. Doesn't look like Circle is running.

**Review by rossf7 — COMMENTED** [review]
Looks good just a query on line endings and automation.

File: helm/kube-eagle/templates/_helpers.tpl (hunk 1 above)
Comment (QuentinBisson) [reply helm/kube-eagle/templates/_helpers.tpl]: You are right, I forgot to enable the project on circle ci side

Comment (QuentinBisson) [reply helm/kube-eagle/templates/_helpers.tpl]: This is fixed and I added the abs config file thanks to @ubergesundheit :D

File: .abs/main.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+replace-app-version-with-git: true
```
Comment (TheoBrigitte) [inline .abs/main.yaml:1]: What is this file for?

File: README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
-[![CircleCI](https://circleci.com/gh/giantswarm/{APP-NAME}-app.svg?style=shield)](https://circleci.com/gh/giantswarm/{APP-NAME}-app)
+[![CircleCI](https://circleci.com/gh/giantswarm/kube-eagle-app.svg?style=shield)](https://circleci.com/gh/giantswarm/kube-eagle-app)
```
Comment (TheoBrigitte) [inline README.md:1]: You need to create a status api token to get access the image right.

https://app.circleci.com/settings/project/github/giantswarm/kube-eagle-app/api

**Review by TheoBrigitte — COMMENTED** [review]
Charts looks alright.

File: .abs/main.yaml (hunk 2 above)
Comment (QuentinBisson) [reply .abs/main.yaml:1]: This is part of the new manage app tooling (see https://github.com/giantswarm/app-build-suite)

File: README.md (hunk 3 above)
Comment (QuentinBisson) [reply README.md:1]: Are you sure as this is not configured for the prometheus-operator app https://app.circleci.com/settings/project/github/giantswarm/prometheus-operator-app/api?return-to=https%3A%2F%2Fcircleci.com%2Fdashboard

File: .abs/main.yaml (hunk 2 above)
Comment (TheoBrigitte) [reply .abs/main.yaml:1]: ok fine

File: README.md (hunk 3 above)
Comment (TheoBrigitte) [reply README.md:1]: prometheus-operator-app is a public repo, but kube-eagle-app is private so you need the token for this to work.

You can check this here: https://github.com/giantswarm/kube-eagle-app/blob/db6357addbba9d1a09e57fa074bcf7ccb0912919/README.md

Comment (QuentinBisson) [reply README.md:1]: I switched it public as this is an app and it should be public

Comment (TheoBrigitte) [reply README.md:1]: :ok_hand:

**Review by TheoBrigitte — APPROVED** [review]
I'm not too deep into this specific app but chart looks alright.

**Pull request merged**